### ISSUE TITLE
ListTile does not show built-in drag handles when using ReorderableListView

### ISSUE THREAD

**Issue by idkq** [flutter/flutter#80553]
## Steps to Reproduce

Copy this example
https://api.flutter.dev/flutter/material/ReorderableListView-class.html#material.ReorderableListView.1

**Expected results:** 
<img width="634" alt="image" src="https://user-images.githubusercontent.com/76702881/114957478-210bb900-9e2f-11eb-94b6-921286b028ab.png">

**Actual results:** 
<img width="317" alt="image" src="https://user-images.githubusercontent.com/76702881/114957465-194c1480-9e2f-11eb-9eb2-a5f9b58d5bbf.png">

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 2.2.0-10.1.pre, on macOS 11.2.3 20D91 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.2)
[✓] Connected device (2 available)
```

</details>

**Comment by pedromassangocode** [comment]
I cannot reproduce this issue on master channel. I cannot reproduce this on latest stable, beta and master channel. Are you testing this on mobile or desktop platforms (Desktop/Chrome for desktop)?

FYI, the draggable handler are only displayed on Desktop platforms.

![Screen Shot 2021-04-16 at 08 32 58](https://user-images.githubusercontent.com/67424416/114988552-5bf10980-9e8e-11eb-8dbd-000c9f025743.png)

See https://api.flutter.dev/flutter/material/ReorderableListView/buildDefaultDragHandles.html

**Comment by idkq** [comment]
Oh ok. I would never had realized that in only shows on desktop.

> Are you testing this on mobile or desktop platforms (Desktop/Chrome for desktop)?

I was doing it on both.

> FYI, the draggable handler are only displayed on Desktop platforms.

Understood, that was the problem. I wasn't aware of this, just started with `ReorderableListView`. We need to add to the `ReorderableListView` documentation.

**Comment by pedromassangocode** [comment]
Hi @idkq 
I think it is already documented in the `buildDefaultDragHandles`  property's documentation (see https://api.flutter.dev/flutter/material/ReorderableListView/buildDefaultDragHandles.html).

**Comment by idkq** [comment]
@pedromassangocode It is, but I personally would add it to the main class since this is the most important feature of the class (which is how you actually drag items to reorder). 

If one does not look into that property specifically they won't know how to manage it. In my case I thought this was a bug because I copied the code from the example and pasted on a new project and it did not work as I expected, but there was no note on what I should expect.

If the example render different results in different platforms I believe it should be noted.

Also the name of the property is a bit confusing, because 'dragHandles' indicates that a 'handle' exist. I understand that on mobile the whole tile is the 'handle' but it is a bit of stretch on taxonomy. A handle is something visible (to open the door for example). Invisible handle is confusing. So I tried to set this property to true and did not see any handle. I wasted 1+ hour or so on this.

It is a great widget but these minor explanations can make a huge difference.
